### DISCUSSION TITLE
Is it possible to conduct Cypress Parallelization without a dashboard account?

### DISCUSSION THREAD

**Discussion by adamalston** [cypress-io/cypress#15916] [General]
I looked through the [parallelization docs](https://docs.cypress.io/guides/guides/parallelization) and all of the commands where parallelization take place seem to use a command that would only work with the dashboard?

Further, when I try to run tests in parallel locally, I get the following error message:
```bash
The --parallel flag you pass was: true

These flags can only be used when recording to the Cypress Dashboard service.
```

**Comment by bahmutov** [comment]
Unfortunately no: parallelization requires the Cypress Test Runner to contact the Cypress API so that each test runner could get the spec to execute based on the historical spec durations which makes the entire test run faster.

Parallelization is just one of the many services the Cypress Dashboard provides, see https://on.cypress.io/dashboard-introduction, and working out how to do this without the central private storage of info is impossible. 

But of course, if you want to invest your own time and set up a system for just running the specs on multiple machines, you could do that. For example, for GitLab CI runners you could regenerate the config file every time there is a new spec file added: https://github.com/kensho/multi-cypress You could devise a similar system for any CI I think. It is up to you to decide if this is a good investment of your development time vs getting the full-featured Dashboard.
